tests/functional/aws-node-sdk/lib/utility/test-utils.js

```diff
@@ -1,8 +1,16 @@
 const { config } = require('../../../../../lib/Config');
 
 const isCEPH = process.env.CI_CEPH !== undefined;
+const isS3C = process.env.S3_END_TO_END !== undefined;
 const itSkipCeph = isCEPH ? it.skip : it;
+const itSkipS3C = isS3C ? it.skip : it;
+const itSkipNotS3C = isS3C ? it : it.skip;
+const itSkipCephS3C = isCEPH || isS3C ? it.skip : it;
+
 const describeSkipIfCeph = isCEPH ? describe.skip : describe.skip; // always skip
+const describeSkipIfAWS = process.env.AWS_ON_AIR ? describe.skip : describe;
+const describeSkipIfS3C = isS3C ? describe.skip : describe;
+
```

Comment:
we have a single shared code base : so we should not skip based on the "product", but only based on the features which are enabled in the target instance of cloudserver (i.e. based on the config.yaml used by that cloudserver)